gamatrix-gog.py

```diff
@@ -265,50 +278,34 @@ def build_config(args):
         config["db_list"].append(
             "{}/{}".format(config["db_path"], config["users"][userid]["db"])
         )
-        # Convert CIDRs into IPv4Network objects; if there are none, disable uploads
```

Comment:
Where did this block go?

Reply:
I missed it in my `git --rebase` flow, apologies! Yeah I really hate it when I let PRs get too big like this. Thank you for catching it.